python/ray/_private/usage/usage_lib.py

```diff
@@ -759,6 +768,7 @@ def generate_report_data(
         extra_usage_tags=get_extra_usage_tags_to_report(gcs_client),
         total_num_nodes=get_total_num_nodes_to_report(gcs_client),
         total_num_running_jobs=get_total_num_running_jobs_to_report(gcs_client),
+        glibc_version=cluster_metadata.get("glibc_version"),
```

Comment:
Set default value to ""?

Reply:
I think the type is: `        "glibc_version": {"type": ["null", "string"]},`, so shouldn't it be `None`?

Reply:
Oh, I see. It will be converted to json null.